### DISCUSSION TITLE
How to pass JSON field through active interaction

### DISCUSSION THREAD

**Discussion by nirname** [AaronLasseigne/active_interaction#551] [Q&A]
I have a json field in my model and would like to pass it as json without type coercion.

Like this:

```ruby
json :content
```
Instead of this:

```ruby
string :content
```

How to do that?

**Comment by AaronLasseigne** [comment]
Could you elaborate a bit on what you're looking for? Ruby doesn't have a `JSON` instance type by default. What classes are you hoping to pass as an input and what are you wanting it to be inside the interaction?
